include/ui/output-component.hpp

```diff
@@ -20,9 +20,12 @@
 #define ERAGPSIM_UI_OUTPUTCOMPONENT_HPP
 
 #include <QDebug>
+#include <QImage>
 #include <QQmlContext>
 #include "core/memory-access.hpp"
 
+// class QImage;
```

Comment:
Why do you include this header? It seems like a forward declaration would be fine here.